src/main/java/seedu/address/model/Model.java

```diff
@@ -114,6 +114,16 @@
      */
     void deleteSaleTag(Tag target);
 
+    /**
+     * Lists all contacts associated with {@code target} tag.
```

Comment:
Is this description accurate? I think the javadocs you have in `AddressBook.java` would be better here! 

Based on the name of the method, I am inclined to think that this method returns a list of contacts. Maybe you want to rename it so it's clearer? Idk. But I think updating the javadocs would be good!

Same for `findBySaleTag`